datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVHelper.scala

```diff
@@ -138,6 +145,17 @@ object MVHelper {
       }
       parentTablesList.add(mainCarbonTable.get)
     }
+
+    // Check if load is in progress in any of the parent table mapped to the datamap
```

Comment:
why this validation is added for timeseries?? Or some bug which u have fixed as a part of this PR

Reply:
This is a bug fixed as part of this PR